content/en/platform/corda/5.0-beta/corda-cli-reference/cli-commands/rpc-permission-templates.md

```diff
@@ -0,0 +1,98 @@
+---
+date: '2023-01-10'
+menu:
+  corda-5-beta:
+    identifier: corda-5-beta-rpc-permissions-templates
+    weight: 6000
+    parent: corda-5-cli-reference
+section_menu: corda-5-beta
+title: "RPC permission templates"
+---
+
+## Create Common RPC Permission Templates
+
+Currently, when the C5 Cluster starts there is a single "admin" account auto-created which can do everything. While this account can be used to perform any action, there is room for error if not used carefully. The idea is to create fine-grained roles for specific actions such as:
+
+• A dedicated role which can create users, roles, permissions and drive all the associations between them;
```

Comment:
Probably needs an extra empty line or else it is rendered like so:
![image](https://user-images.githubusercontent.com/31008341/212139655-ad881721-9d7d-492f-9846-805742228ef9.png)


```suggestion
• A dedicated role which can create users, roles, permissions and drive all the associations between them;

```